app/routes/pages/components/LandingPage.js

```diff
@@ -209,4 +276,33 @@ const committeeEmails = [
   }
 ];
 
+const socialMedia = {
+  facebook: [
+    {
+      link: 'https://www.facebook.com/AbakusNTNU/?ref=bookmarks',
+      textInfo: 'Hovedside for Abakus'
+    },
+    {
+      link:
+        'https://www.facebook.com/Abakus-Bedriftsarrangementer-245005099523282/',
+      textInfo: 'Bedriftsarrangementer'
+    }
+  ],
+  instagram: [
+    {
+      link: 'https://www.instagram.com/abakusntnu/',
+      textInfo: 'Abakus instagram'
```

Comment:
Same as beneath, "Abakus offisiell - @abakusntnu" perhaps? 

(proof beneath hehe)
![screenshot from 2019-03-05 00-05-46](https://user-images.githubusercontent.com/25204035/53768955-aabfcd00-3eda-11e9-89c7-a7cfae99e439.png)

Reply:
"Abakus offisiell - @abakusntnu" is a bit too long, so it doesn't render well on desktop, does "Offisiell - @abakusntnu" work?